app/invitations/controllers.py

```diff
@@ -123,6 +124,59 @@ def send_request(new_user, committee):
         db.session.rollback()
         return Response.RequestError
 
+##
+## @brief      Sends an invitation to a committee-head to close
+##             a charge.
+##
+## @param      committee       The committee to join.
+## @param      new_user        The user to be invited.
+##
+## @return     True if email sent, False if not.
+##
+def send_close_request(user, committee, chargeID):
+    
+    admins = db.session.query(Users).filter(Users.is_admin == True).all()
+
+    admin_emails = []
+    for user in admins:
+        admin_emails.append(user.id + "@rit.edu")
+
+    invite = and_(
+        Invitations.user_name == committee.head,
+        Invitations.committee_id == committee.id,
+        Invitations.isInvite == False
+    )
+    
+    invitation = Invitations (
+        user_name= user.id,
+        committee= committee,
+        committee_id = committee.id,
+        charge_id = chargeID,
+        isInvite=False
+    )
+    
+    try:
+        db.session.add(invitation)
+        db.session.commit()
+        email = {}
+        email["title"] = "Close Charge Request"
+        email["sender"]=("SG TigerTracker", "sgnoreply@rit.edu")
+        email["recipients"] = admin_emails
+        email["subtype"] = "related"
+        email["html"] = render_template(
+            'close_charge_request.html',
+            user_name= committee.head,
+            charge_name= chargeID,
+            time_stamp= time.time(),
+            request_url= app.config['CLIENT_URL'] + str(invitation.id)
```

Comment:
I looked into what happens when if a user clicks on this request URL.  They get directed to the page that uses the `Invitation` component that is intended to be displayed to new users of TigerTracker. 

Because of this I think that the request URL should instead be routing directly to the charge that needs to be closed. This solution would require the least refactoring (no need to mess with the current invitation page).

That being said I'm not sure what a "clean" way to get the current URL. `CLIENT_URL` is hardcoded as `<hostname>/invitation/`, which is different across environments (`tigertracker.rit.edu`, `tigertrackerstage.rit.edu`) and used in a couple different places.

There's some wonky stuff with the Invitation page and adding members to a committee that I think need to be brought up in the meeting. It feels like this functionality has been forgotten about and we need to figure out what to do with it.